cmd/kubeadm/app/constants/constants.go

```diff
@@ -472,6 +472,7 @@ var (
 		27: "3.5.10-0",
 		28: "3.5.10-0",
 		29: "3.5.10-0",
```

Comment:
[etcd to 3.5.11](https://github.com/etcd-io/etcd/blob/main/CHANGELOG/CHANGELOG-3.5.md): fixes distributed tracing, OpenTel CVE. 

Should we bump etcd versions for v1.29 and early release as well?

Reply:
we can bump an old kubeadm version as well, but that should be only 1.29, because kubeadm supports skew against control-plane N and N-1 only.

Reply:
...but i guess bumping all of the 3.5.x versions is less confusing.